app/models/katello/host/subscription_facet.rb

```diff
@@ -242,33 +240,25 @@ def remove_subscriptions(pools_with_quantities)
         ForemanTasks.sync_task(Actions::Katello::Host::RemoveSubscriptions, self.host, pools_with_quantities)
       end
 
-      def self.find_host(facts, organization)
+      def self.ensure_new_host(facts, organization)
         host_name = propose_existing_hostname(facts)
         uuid_fact_id = RhsmFactName.find_by(name: 'dmi::system::uuid')&.id || -1
         hosts = ::Host.unscoped.distinct.left_outer_joins(:fact_values)
```

Comment:
Extra credit:  But can this be moved to a method in Host model via Host:::Managed extensions

Reply:
Might be a good idea. Let me think about that over lunch ;)

Reply:
I'd rather skip this as I don't see much benefit - but I do see why you would suggest it. Perhaps down the road a larger subset of subscription facet methods could be moved to Managed extensions since some do relate less to candlepin